migrations/1624695890253-CreateRfqmWorkerHeartbeatTable.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const TABLE_NAME = 'rfqm_worker_heartbeats';
+
+const rfqmWorkerHeartbeatTable = new Table({
+    name: TABLE_NAME,
+    columns: [
+        { name: 'address', type: 'varchar', isPrimary: true },
+        { name: 'timestamp', type: 'timestamptz', default: 'NOW()' },
+        { name: 'balance', type: 'numeric' },
+        { name: 'index', type: 'int' },
+    ],
+});
+
+export class CreateRfqmWorkerHeartbeatTable1624695890253 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.createTable(rfqmWorkerHeartbeatTable);
+        // Update timestamp on every write
+        await queryRunner.query(`
+            CREATE OR REPLACE FUNCTION update_timestamp_column()
```

Comment:
Was this created by the TypeORM program, or was this created by you?

Reply:
Me

https://www.revsys.com/tidbits/automatically-updating-a-timestamp-column-in-postgresql/